STYLE.md

```diff
@@ -3,7 +3,8 @@
 * The Envoy source code is formatted using clang-format. Thus all white spaces, etc.
   issues are taken care of automatically. The CircleCI tests will automatically check
   the code format and fail. There are make targets that can both check the format
-  (check_format) as well as fix the code format for you (fix_format).
+  (check_format) as well as fix the code format for you (fix_format). To run
+  these checks locally, see [Support Tools](support/README.md).
```

Comment:
Since you're here, can you also add a mention about clang-tidy?

Errors in [.clang-tidy](
https://github.com/envoyproxy/envoy/blob/master/.clang-tidy#L4) are enforced while other warnings are suggestions.